lib/rabbit/command/rabbit-slide.rb

```diff
@@ -772,7 +782,12 @@ def generate_slide
       end
 
       def slide_path
-        "#{@data.slide_conf.base_name}.#{slide_source_extension}"
+        case @data.author_conf.markup_language
+        when :pdf
+          File.join("pdf", "#{@data.slide_conf.id}-#{@data.slide_conf.base_name}.pdf")
```

Comment:
Note: It would be better to unify this with the following, but I can't think of a good way to do it.

https://github.com/rabbit-shocker/rabbit/blob/01885b247a230b4b122831cd48d5d8935b759879/lib/rabbit/task/slide.rb#L203-L205

Reply:
Thanks! Nice idea! Fixed!